### DISCUSSION TITLE
Adonis website firewalls my ISP/AS again

### DISCUSSION THREAD

**Discussion by baybal** [adonisjs/core#1643] [General]
Hello,

It seems that cloudflare is firewalling my AS again. Can you add AS35104 to the exemptions list for your cloudflare setup?

**Comment by baybal** [comment]
@RomainLanz

**Reply by baybal** [reply]
@thetutlage

**Reply by thetutlage** [reply]
Mind sharing more info. I have no idea what an `AS` is and also how to go about whitelisting it. So please provide more info or maybe an article from cloudflare

**Reply by baybal** [reply]
AS is a list of IP prefixes belonging to a network.

Here is that of my ISP with which I have problems https://ipinfo.io/AS35104

And here is a nice videoguide how to do whitelist in cloudlfare https://youtu.be/Sz9tBeyv2x0?t=413

**Reply by baybal** [reply]
@thetutlage

**Reply by thetutlage** [reply]
Want to give it a try now?

**Reply by baybal** [reply]
@thetutlage seem to work now! Thanks!

**Reply by baybal** [reply]
@thetutlage the part of my ISP network that uses another AS seems to still be blocked AS9198 https://ipinfo.io/AS9198
